arkouda_workflows/delete-arkouda-on-kubernetes-cronworkflow.yaml

```diff
@@ -0,0 +1,185 @@
+            template: delete-service-role
+          - name: delete-service-role-binding
+            template: delete-service-role-binding
+          - name: delete-metrics-exporter-service
+            template: delete-metrics-exporter-service
+        
+    - name: delete-locale-deployment
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: apps/v1
+          kind: Deployment
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-locale
+            labels:
+              app: {{ workflow.parameters.arkouda-instance-name }}-locale
+
+    - name: delete-locale-headless-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-locale-headless
+
+    - name: delete-locale-ssh-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-locale-ssh
+
+    - name: delete-server-deployment
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Pod
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-server
+            labels:
+              app: {{ workflow.parameters.arkouda-instance-name }}-server
+
+    - name: delete-server-headless-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-server-headless
+
+    - name: delete-server-ssh-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-server-ssh
+
+    - name: delete-arkouda-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}
+
+    - name: delete-arkouda-metrics-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-metrics
+
+    - name: delete-metrics-exporter-service
+      resource:
+        action: delete
+        manifest: |
+          apiVersion: v1
+          kind: Service
+          metadata:
+            name: {{ workflow.parameters.arkouda-instance-name }}-metrics-exporter
+
+    - name: delete-arkouda-server-launch-script 
+      resource:
+        action: delete
+        manifest: |
+          kind: ConfigMap
+          apiVersion: v1
+          metadata:
+            name: arkouda-server-launch-script
```

Comment:
@hokiegeek2 this is the only one that doesn't use `{{ workflow.parameters.arkouda-instance-name }}` just double checking that's inteded